File: backend/src/main/java/com/bomnuocv1/domain/entity/DashboardSummary.java
package com.bomnuocv1.domain.entity;

import lombok.Builder;
import lombok.Getter;

import java.math.BigDecimal;
import java.util.List;

@Getter
@Builder
public class DashboardSummary {

    private final BigDecimal totalUncollectedDebt;
    private final String formattedTotalDebt;
    private final long todayPumpCount;
    private final List<RecentTransaction> recentTransactions;
}

File: frontend/app/src/main/java/com/vn/bomnuocv1/domain/model/DashboardSummary.kt
package com.vn.bomnuocv1.domain.model

import java.math.BigDecimal

data class DashboardSummary(
    val totalUncollectedDebt: BigDecimal,
    val formattedTotalDebt: String,
    val todayPumpCount: Long,
    val recentTransactions: List<RecentTransaction>
)
